packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -211,13 +211,14 @@ describe('HttpServer (integration)', () => {
     host?: string;
   }): HttpServer {
     const options: HttpServerOptions = {protocol: 'https', host};
+    const certDir = path.resolve(__dirname, '../../../fixtures');
```

Comment:
This is what annoys me.The directory structure is different between source code and the distribution. In this case, developers can be confused as they might assume `../../../fixtures` resolve to `packages/fixtures` based on the source code directory chain.